libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -977,7 +983,7 @@ protected Void doInBackground(Void... nothings) {
 	        try {
 	        	OAuth2.revokeRefreshToken(HttpAccess.DEFAULT, new URI(loginServer), refreshToken);
 	        } catch (Exception e) {
-	        	Log.w("SalesforceSDKManager:revokeToken", e);
```

Comment:
Yup - and other logging enhancements. That's what our `logging improvements` story on the backlog is for ;-)